easybuild/easyconfigs/m/mii/mii-1.1.0-GCCcore-9.3.0.eb

```diff
@@ -24,20 +26,26 @@ skipsteps = ['configure', 'build']
 installopts =  'PREFIX=%(installdir)s/ '
 installopts += 'MII_ENABLE_SPIDER=yes '
 
-postinstallcmds = ['mkdir -p %(installdir)s/data/']
+_arch = os.getenv("RSNT_ARCH")
```

Comment:
local variables in easyconfig should be named `local_<something>`. I'm not sure what is the impact of having variables starting with `_`.

Reply:
starting variables with `_` is part of the [recommended naming scheme for local variables in easyconfig files](https://docs.easybuild.io/en/latest/Easyconfig-files-local-variables.html#recommended-naming-scheme-for-local-variables-in-easyconfig-files). I can change it to use `local_` instead, but it is supported by EB.

Reply:
Ok then. You teach me something about EasyBuild :D